dist/dropdown/ds6/dropdown.css

```diff
@@ -55,40 +63,69 @@ button.fake-menu__item {
   width: 100%;
 }
 div.menu__item[role^="menuitem"] span:not([class*="menu__status"]),
+div.listbox__option[role^="option"] span:not([class*="menu__status"]),
 a.fake-menu__item span:not([class*="menu__status"]),
-button.fake-menu__item span:not([class*="menu__status"]) {
+button.fake-menu__item span:not([class*="menu__status"]),
+div.menu__item[role^="menuitem"] span:not([class*="listbox__status"]),
+div.listbox__option[role^="option"] span:not([class*="listbox__status"]),
+a.fake-menu__item span:not([class*="listbox__status"]),
+button.fake-menu__item span:not([class*="listbox__status"]) {
   overflow: hidden;
   white-space: nowrap;
   text-overflow: ellipsis;
 }
 div.menu__item[role^="menuitem"]:focus,
+div.listbox__option[role^="option"]:focus,
 a.fake-menu__item:focus,
 button.fake-menu__item:focus,
 div.menu__item[role^="menuitem"]:hover,
+div.listbox__option[role^="option"]:hover,
 a.fake-menu__item:hover,
 button.fake-menu__item:hover {
-  background-color: #f5f5f5;
+  background-color: #e1fbff;
+  color: #006efc;
 }
 div.menu__item[role^="menuitem"]:focus,
+div.listbox__option[role^="option"]:focus,
 a.fake-menu__item:focus,
 button.fake-menu__item:focus {
   outline: 1px solid #c7c7c7;
 }
 div.menu__item[role^="menuitem"]:active,
+div.listbox__option[role^="option"]:active,
 a.fake-menu__item:active,
 button.fake-menu__item:active {
-  font-weight: bold;
+  background-color: #10299c;
+  color: #e1fbff;
+}
+div.menu__item[role^="menuitem"]:active .menu__status,
+div.listbox__option[role^="option"]:active .menu__status,
+a.fake-menu__item:active .menu__status,
+button.fake-menu__item:active .menu__status,
+div.menu__item[role^="menuitem"]:active .listbox__status,
+div.listbox__option[role^="option"]:active .listbox__status,
+a.fake-menu__item:active .listbox__status,
+button.fake-menu__item:active .listbox__status,
+div.menu__item[role^="menuitem"]:active .fake-menu__status,
+div.listbox__option[role^="option"]:active .fake-menu__status,
+a.fake-menu__item:active .fake-menu__status,
+button.fake-menu__item:active .fake-menu__status {
+  color: #e1fbff;
 }
 div.menu__item[role^="menuitem"]:not(:last-child),
+div.listbox__option[role^="option"]:not(:last-child),
 a.fake-menu__item:not(:last-child),
 button.fake-menu__item:not(:last-child) {
   margin-bottom: 1px;
 }
-div.menu__item[role^="menuitem"][aria-checked="true"] .menu__status {
+div.menu__item[role^="menuitem"][aria-checked="true"] .menu__status,
+div.listbox__option[role^="option"][aria-selected="true"] .menu__status,
+div.menu__item[role^="menuitem"][aria-checked="true"] .listbox__status,
+div.listbox__option[role^="option"][aria-selected="true"] .listbox__status {
   visibility: visible;
 }
 a.fake-menu__item {
-  color: #111820;
+  color: #006efc;
```

Comment:
I don't think we should be changing the menu items from gray to blue. This is the spec I went off:

![screen shot 2017-12-06 at 4 26 13 pm](https://user-images.githubusercontent.com/38065/33692459-44188bca-daa2-11e7-9e7b-4f0765ba144d.png)

See https://github.com/eBay/skin/issues/10

This is frustrating I know because the DSL still shows the old blue menu items. We desperately need versions on these designs!

Reply:
Wow, the design on DSL is old? It looked new.

Reply:
This is what I went off of: https://dsl.ebay.com/ds6/buttons

![image](https://user-images.githubusercontent.com/105656/34547856-5ca88fc6-f0bb-11e7-8fc7-f91d6c9328d4.png)

They specifically talk about menus _and_ "select" form elements. Which is right?

Reply:
I'm seeking confirmation from design. If I don't get any confirmation in a reasonable time, I'll revert the changes.

Reply:
Please for now just use the same spec that I used for menu. That spec supersedes the one on DSL. Trust me ;-)

When we get even newer specs, we can then update menu and listbox at the same time.

Reply:
>I'm seeking confirmation from design. If I don't get any confirmation in a reasonable time, I'll revert the changes.

And ... I got a response. I'll revert.